tools/ci_build/github/azure-pipelines/linux-gpu-tensorrt-packaging-pipeline.yml

```diff
@@ -0,0 +1,43 @@
+variables:
+  PackageName: 'Microsoft.ML.OnnxRuntime.Gpu.Tensorrt'
+
+jobs:
+
+- job: Linux_C_API_Packaging_GPU_TensorRT_x64
+  workspace:
+    clean: all
+  timeoutInMinutes:  120
+  pool: 'Onnxruntime-Linux-GPU'
+  variables:
+    CUDA_VERSION: '11.1'
```

Comment:
BTW, I suggest we change all the other inference GPU pipelines to CUDA 11.1. Now they are 11.0. But TRT and training are both using 11.1. If we make them the same, it looks simpler and cleaner.

Reply:
this pipeline will initially be used for internal, so we have flexibility with the versions but
yes, it would be good to have consistency.
easy to do for Linux/docker but more work for Windows right?